Code/Editor/AzAssetBrowser/AzAssetBrowserWindow.cpp

```diff
@@ -692,6 +692,7 @@ void AzAssetBrowserWindow::SetOneColumnMode()
         }
         m_ui->m_thumbnailView->SetThumbnailActiveView(false);
         m_ui->m_expandedTableView->SetExpandedTableViewActive(false);
+        m_ui->m_searchWidget->setDisabled(false);
```

Comment:
nit: would use `setEnabled(true)` for readability.

Reply:
sounds good, changed it